### DISCUSSION TITLE
PKI

### DISCUSSION THREAD

**Discussion by Taju8704** [Keyfactor/ejbca-ce#425] [Q&A]
Team, have an issue with our document signer 
failing to open the administration web link in the web interface (browser) in order to access and activate workers. Normally, when we try to open this link it prompts you to enter S Admin 3 times, but now it's only asking once. I'm suspecting another colleague used a wrong password several times before discovering the password was wrong. Greatly appreciate any feedback on this. Have attached some photos for reference.

**Comment by primetomas** [comment]
I think you are aiming this question to https://github.com/Keyfactor/signserver-ce/

**Reply by Taju8704** [reply]
Yes yes yes

**Comment by primetomas** [comment]
Good, closing this thread then.
